src/bytes.rs

```diff
@@ -12,234 +12,3 @@ macro_rules! concat_bytes {
         }
     }
 }
-
-#[macro_export]
-macro_rules! bytes_wrapper {
```

Comment:
I used this macro to define the `ProofOfIndexing` type in the `graph-ateway` crate (file: `poi.rs). Removing it from here would impact the gateway PIO blocking functionality. 

The simplest solution for this I can think of right now is to expand the macro "manually".

Reply:
It looks like `ProofOfIndexing` is just a 32-byte hash. So it could just be replaced by `ethers::types::H256` or `ethers::abi::Hash`.

Reply:
Hmmm, wrapping one of those types into a "new type" would be an option to have a semantically distinct type 👍🏼